File: Design Patterns - MVC Pattern (12 Samples for Beginners)/[C#]-Design Patterns - MVC Pattern (12 Samples for Beginners)/C#/MVCDemo3Views/Controllers/HelloWorldController.cs
using System.Web.Mvc;

namespace MVCDemo3Views
{
    public class HelloWorldController : Controller
    {
        public ActionResult SayHello()
        {
            return View();            

        }

        //[ChildActionOnly]
        public ActionResult SayPartialHello()
        {
            return PartialView();
        }
    }
}

File: Intro to ASP.NET MVC 4/[VB]-Intro to ASP.NET MVC 4/VB/MvcMovie/Controllers/HelloWorldController.vb
Namespace MvcMovie
    Public Class HelloWorldController
        Inherits System.Web.Mvc.Controller

        '
        ' GET: /HelloWorld
        Function Index() As ActionResult
            Return View()
        End Function

        '
        ' GET: /HelloWorld
        Function Welcome(ByVal name As String, Optional ByVal numTimes As Integer = 1) As ActionResult
            ViewData("Message") = "Hi " & name
            ViewData("NumTimes") = numTimes
            Return View()
        End Function

    End Class
End Namespace